src/scales/scale.linearbase.js

```diff
@@ -23,16 +25,26 @@ function generateTicks(generationOptions, dataRange) {
 	var precision = generationOptions.precision;
 	var spacing, factor, niceMin, niceMax, numSpaces;
 
+	var rmin = dataRange.min;
+	var rmax = dataRange.max;
+	var isNullOrUndef = helpers.isNullOrUndef;
```

Comment:
I would move this alias globally, as done in #5937 ([comment](/chartjs/Chart.js/pull/5937#discussion_r245364972)).